Update "Deploy Automatically" for 'make deploy' (#653)
submariner-io/submariner-website: src/content/development/shipyard/advanced/_index.en.md

  make deploy DEPLOY_ARGS='--deploytool operator --deploytool_submariner_args "--cable-driver wireguard"'
  ```

As shown above, arguments can be passed directly to the Broker and Submariner.

> The deploy script also has some additional flags that can be set and the script will set the Broker and Submariner arguments properly for you (see the flags defined in [deploy.sh](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/deploy.sh) for the full list):

Reviewer: Took me a minute to figure out what you meant here, this might be more clear:

```suggestion
The deploy script also has flags that group common options together for easier UX. For example, the [`service_discovery` flag in `deploy.sh`](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/deploy.sh) will handle the `--components service-discovery,connectivity` flags mentioned above.
```

Reviewer: Nice edit in the latest version, thanks @Billy99!

* **globalnet:** When set, deploys Submariner with the globalnet controller, and assigns a unique Global CIDR to each cluster.

  ```bash
  make deploy DEPLOY_ARGS='--globalnet'
  ```

> * **cable_driver:** When set, deploys Submariner with the input cable driver, which configures the tunneling method for connections between clusters.

Reviewer: ```suggestion
* **cable_driver:** Override the default cable driver to configure the tunneling method for connections between clusters.
```
